frontend/app/controllers/TierController.scala

```diff
@@ -87,7 +87,7 @@ trait UpgradeTier extends StripeServiceProvider with CatalogProvider {
       PageInfo(initialCheckoutForm = formI18n, stripePublicKey = stripeKey)
     }
 
-    def fromFree(subscription: FreeSubscription, contact: Contact[FreeTierMember, NoPayment]): Future[Result] =
+    def fromFree(subscription: FreeSubscription, contact: Contact[_, _]): Future[Result] =
```

Comment:
I'm using underscores here to force compile errors if you try to do anything with the type parameters as I want to remove them soon